DESCRIPTION

```diff
@@ -2,55 +2,51 @@ Package: pegboard
 Title: Explore and Manipulate Markdown Curricula from the Carpentries
 Version: 0.5.3
 Authors@R: c(
-    person(given = "Zhian N.",
-           family = "Kamvar",
-           role = c("aut", "cre"),
-           email = "zkamvar@carpentries.org",
+    person("Zhian N.", "Kamvar", , "zkamvar@carpentries.org", role = c("aut", "cre"),
            comment = c(ORCID = "0000-0003-1458-7108")),
-    person(given = "Toby",
-           family = "Hodges",
-           role = c("ctb"),
-           email = "tobyhodges@carpentries.org"),
-    person())
-Description: The Carpentries (<https://carpentries.org>) curricula is made of of
-    lessons that are hosted as websites. Each lesson represents between a half
-    day to two days of instruction and contains several episodes, which are
-    written as 'kramdown'-flavored 'markdown' documents and converted to HTML
-    using the 'Jekyll' static website generator. This package builds on top of
-    the 'tinkr' package; reads in these markdown documents to 'XML' and
-    stores them in R6 classes for convenient exploration and manipulation of
-    sections within episodes.
+    person("Toby", "Hodges", , "tobyhodges@carpentries.org", role = "ctb")
+  )
```

Comment:
Could you do two things here:

1. change the author list back to its original formatting? The formatter inserted an extra comma and this formatting lends itself well to copy + paste + modify so you don't have to add or remove a comma when you add a new person. 
2. add your own name as a contributor

```
Authors@R: c(
    person(given = "Zhian N.",
           family = "Kamvar",
           role = c("aut", "cre"),
           email = "zkamvar@carpentries.org",
           comment = c(ORCID = "0000-0003-1458-7108")),
    person(given = "Toby",
           family = "Hodges",
           role = c("ctb"),
           email = "tobyhodges@carpentries.org"),
    person())
```